mobile/android/android-device-file-system.ts

```diff
@@ -45,8 +47,10 @@ export class AndroidDeviceFileSystem implements Mobile.IDeviceFileSystem {
 		return (() => {
 			this.adb.executeCommand(["push", projectFilesPath, deviceAppData.deviceProjectRootPath]).wait();
 
-			let command = _.map(localToDevicePaths, (localToDevicePathData) => localToDevicePathData.getDevicePath()).join(" ");
-			this.adb.executeCommand(["chmod", "0777", command]).wait();
+			let command = _.map(localToDevicePaths, (localToDevicePathData) => '"' + localToDevicePathData.getDevicePath() + '"').join(" ");
```

Comment:
You can optionally use a template string here
```Typescript
`"${localToDevicePathData.getDevicePath()}"`
```

Would be a bit clearer I think - definately not a merge-stopper